packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -109,6 +107,8 @@ export function getValidatorApi({
   const MAX_API_CLOCK_DISPARITY_SEC = Math.min(0.5, config.SECONDS_PER_SLOT / 2);
   const MAX_API_CLOCK_DISPARITY_MS = MAX_API_CLOCK_DISPARITY_SEC * 1000;
 
+  const defaultGraffiti = getDefaultGraffiti();
```

Comment:
this doesn't seem the right place to me to call this, caching this is definitely they way to go as we don't wanna add request latency during block proposal but this needs to be handled differently imo.

Potential solution could be that have a period job (e.g. every epoch) to call this api and update the cached graffiti. I would expect this request to be rather cheap as well and this ensure the information is correct and up-to-date / independent from when Lodestar vs. EL was updated, or last time restarted.